libuser/src/task_start.S

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) 2021 by Rivos Inc.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+
+.option norvc
+
+.section .text.start
+
+// The entry point for a task.
+.global _start
+_start:
+
+.option push
+.option norelax
+    la gp, __global_pointer$
+.option pop
+    la sp, _stack_end
+
+    la t1, task_main
+    la ra, 1f
+    jr t1
+
+    j 1f
+1:
```

Comment:
Just `call task_main` is sufficient.

Reply:
Ah good catch. Had a note somewhere when I saw your patches to change this too. Thanks!

Fixed.